docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-itorders-roles-proc.adoc

```diff
@@ -1,29 +1,21 @@
-[id='case-management-itorders-roles-proc']
+[id='case-management-itorders-roles-proc-{context}']
 = Creating case roles
 
-.Procedure 
-. *Define the case roles*
+Case roles 
+
+The IT Orders sample case management project includes the following roles:
 +
-Click image:3898.png[] to open the *Properties* menu on the right side of the designer and open the *Editor for Case Roles*. Click *Add Case Role* to add case roles. 
+* `owner` - the employee who is making the hardware order request. There can be only one of these roles.
+* `manager` - the employee's manager; the person who will approve or deny the requested hardware. There is only one manager in the IT Orders project.
```

Comment:
same comment here - manager is a role and not really in the project so it must be assigned when starting a case instance

Reply:
done in 2568 branch.